etc/tests/tokenTest.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
```

Comment:
While this is useful, I'm not sure that tests/ is the right folder for this. After a bit of discussion, Daisie and I found that there's no real spot for us to put helper scripts like this (that aren't part of the build process), so... I think here is fine until we reorganize it.